script/grblcon.py

```diff
@@ -42,6 +42,33 @@ def __str__(self):
         ])
 
 
+class ADCs:
+    """ A simple object to parse and maintain ADC values
+    """
+
+    # pylint: disable=too-few-public-methods
+
+    def __init__(self, adc_string='0,0,0,0,0,0'):
+
+        # pylint: disable=invalid-name
+        self.x, self.y, self.z, self.c, self.force, self.revdiv = [
+            int(x) for x in adc_string.split(',')
+        ]
+
+    def __str__(self):
+        return '\n'.join([
+            'ADCs',
```

Comment:
This function appears never to be used. Is that on purpose? If so, why not delete it? If not, put something in that uses it.